code/game/area/areas/station.dm

```diff
@@ -482,6 +482,7 @@
 	mood_message = "I love being in the bar!"
 	mood_trait = TRAIT_EXTROVERT
 	sound_environment = SOUND_AREA_SMALL_SOFTFLOOR
+	times_eaten_counter = BREAK_ROOM_DESIGNATION
```

Comment:
This works, but it's very weird. I'd prefer if this was an area flag that was checked alongside times_eaten_counter

Reply:
no flags, this needs to be something the crew can apply dynamically to a room, otherwise a constructed dining room never becomes good to eat in